Test/VimWpfTest/CommandLineEditIntegrationTest.cs

```diff
@@ -47,7 +47,8 @@ protected virtual void Create(params string[] lines)
                 VimEditorHost.EditorFormatMapService.GetEditorFormatMap(_vimBuffer.TextView),
                 VimEditorHost.ClassificationFormatMapService.GetClassificationFormatMap(_vimBuffer.TextView),
                 CommonOperationsFactory.GetCommonOperations(_vimBuffer.VimBufferData),
-                _clipboardDevice);
+                _clipboardDevice,
+                false);

```

Comment:
Consider passing the constant by name here so the intent is clear: `isFirstCommandMargin: false`

Reply:
Good idea.

Reply:
Done.